packages/fxa-auth-server/lib/routes/validators.js

```diff
@@ -440,6 +440,13 @@ module.exports.subscriptionsSubscriptionValidator = isA.object({
   subscription_id: module.exports.subscriptionsSubscriptionId
     .required()
     .description(DESCRIPTIONS.subscriptionId),
+  promotion_amount_off: isA
+    .number()
+    .min(0)
+    .max(100)
```

Comment:
Per above description:
```js
  promotionAmountOff:
    'Amount (in the currency specified) that will be taken off the subtotal of any invoices for this customer.',
```
Since this is seemingly an _amount/currency_, not sure if `.max(100)` limit makes sense here (versus `promotion_percent_off` below where percentages are easier to clamp). Otherwise it might mean we can't do more than a $1 or $100 discount (depending on whether the amount off is cents or dollars).